LOAN/LoanContracts/LoanCore.rdf

```diff
@@ -948,23 +882,8 @@
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label>payment history</rdfs:label>
-		<skos:definition>a record of actual payments of principal, interest, and other related amounts made by a borrower to a lender or servicer in order to fulfill their re-payment obligation</skos:definition>
 		<skos:definition>record of actual payments of principal, interest, and other related amounts made by a borrower to a lender or servicer in order to fulfill their re-payment obligation</skos:definition>
-		<fibo-fnd-utl-av:explanatoryNote>Just like a PaymentSchedule except that the dates and amounts are actuals.</fibo-fnd-utl-av:explanatoryNote>
-		<fibo-fnd-utl-av:explanatoryNote>Similar to a PaymentSchedule except that the dates and amounts are actuals</fibo-fnd-utl-av:explanatoryNote>
-	</owl:Class>
-	
-	<owl:Class rdf:about="&fibo-loan-ln-ln;PhoneNumber">
-		<rdfs:subClassOf rdf:resource="&fibo-fnd-plc-adr;VirtualAddress"/>
-		<rdfs:subClassOf>
-			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-fnd-utl-val;hasDataValue"/>
-				<owl:someValuesFrom rdf:resource="&xsd;string"/>
-			</owl:Restriction>
-		</rdfs:subClassOf>
-		<rdfs:label>phone number</rdfs:label>
-		<skos:definition>a virtual address for communication via phone</skos:definition>
-		<skos:editorialNote>Future consideration: move this property to Addresses ontology (fibo-fnd-plc-adr).</skos:editorialNote>
+		<fibo-fnd-utl-av:explanatoryNote>Similar to a PaymentSchedule except that the dates and amounts are actuals.</fibo-fnd-utl-av:explanatoryNote>
```

Comment:
PaymentHistory should be dated. Especially with hasOutstandingAmount (which should be a property of the Loan not the History.
More practically we should be able easily to turn a Schedule into a History.

Reply:
Yes - for another issue